data/process_csv.rb

```diff
@@ -0,0 +1,38 @@
+require 'csv'
```

Comment:
What's the purpose of this script? Could you write a summary for it? When should be used?

Reply:
This is the script which takes apic.csv and splits it into two files, linked by UUID.

Reply:
Could you write a summary in as a Ruby comment at the beginning of the file?